### PR TITLE
I changed my mind about my the most favorite phrase

### PR DESCRIPTION
**Pull request by AleksChernysh**
Я згадав про свої найулюбленіші цитати одного мудрого мудреця (вибачаюсь за тавтологію) з шедевру мультиплікації "Панда кун-фу"

### CODE REVIEW

File: Po.txt
Comment (rodikno) [inline Po.txt:1]: Those are some gems I didn't expect, thanks :D

**Comment by rodikno** [comment]
Good job @AleksChernysh , but please don't forget to resolve the conflicts or otherwise you won't be able to merge it
Tag me in the comments when the conflict is resolved, TY!

**Comment by AleksChernysh** [comment]
@rodikno

**Review by rodikno — APPROVED** [review]
TY, approved

**Pull request merged**